api/app/urls.py

```diff
@@ -15,6 +15,7 @@
     re_path(r"^api/v2/", include("api.urls.v2", namespace="api-v2")),
     re_path(r"^admin/", admin.site.urls),
     re_path(r"^health", include("health_check.urls", namespace="health")),
+    path("healthcheck", include("health_check.urls", namespace="health")),
```

Comment:
Maybe we should add a short comment explaining that this was added to support aptible deployments?

Reply:
Added in e78cf072b130b9312e99c823525b18a89727c621 ✅